### DISCUSSION TITLE
chess has broken icons

### DISCUSSION THREAD

**Discussion by arash-hacker** [lowlighter/metrics#1289] [Q&A]
<img width="441" alt="Screenshot 2022-11-20 at 20 37 50" src="https://user-images.githubusercontent.com/6451804/202922337-71a6f495-942b-4a81-b55f-0dff304af353.png">

**Comment by lowlighter** [comment]
Hi 👋 !

Could you tell me what is your browser/OS combination ?

This occurs when the utf8 chess pawns ♙ gets replaced as the emoji one ♟

**Reply by arash-hacker** [reply]
Chrome  / Macbook Pro (M1 2020). 
<img width="364" alt="Screenshot 2022-11-20 at 22 03 23" src="https://user-images.githubusercontent.com/6451804/202925995-3d8e5a02-e013-46fa-a801-45a8486eca83.png">
In my default rendering for SVG it's ok smh, so it's browser default.
<img width="385" alt="Screenshot 2022-11-20 at 22 04 57" src="https://user-images.githubusercontent.com/6451804/202926062-f7b687a7-8def-415c-bc12-b07fae2f2b64.png">

**Reply by lowlighter** [reply]
https://github.com/lowlighter/metrics/blob/5c9ac90d347dcb4739cfd8563004217109b4e592/source/templates/classic/partials/chess.ejs#L59-L62

Normally the unicode varion selector (`&#xFE0E;`) is supposed to force the render as utf rather than emoji, but according to this [stackoverflow thread](https://stackoverflow.com/questions/32915485/how-to-prevent-unicode-characters-from-rendering-as-emoji-in-html-from-javascrip) it isn't always honored by all browsers...

One suggestion that may be working according to the answers is to use `font-family: monospace` instead so it doesn't use an emoji font.

Could you try editing the svg through your browser and add `font-family: monospace` to `.chess-move` css class and see if it fixes the issue for you ?

**Reply by lowlighter** [reply]
Ok the hack of `font-family: monospace` seems to work (I was able to reproduce on Edge Windows), I pushed it through 
69604e9 👍

**Reply by arash-hacker** [reply]
Thanks for your effort.     
I think it's more flexible and pragmatic if we use SVG as icons too, like below.   
If you agree, I can send PR with SVG pieces.
![](https://raw.githubusercontent.com/arash-hacker/gess/main/chess.svg)

**Reply by lowlighter** [reply]
Yeah I think SVG pieces may be more visually appealing and just more consistent overall, but we need to ensure the ones that will be used are compatible with this project license (i.e. authorized free usage and redistribution or at least explicitely allowed by their owners). If that's the case I think it'd be a nice change 👍  

___

Also I noticed you used chess.com, do you think you could also do a PR to add the support to upstream if it's not too much work ?
It'd be nice to have support other than lichess too 😄

**Reply by arash-hacker** [reply]
1. Sure, SVG with a compatible license.
2. chess.com provider  😄👍
